SoftwareGuide/Latex/Appendices/CodingStyleGuide.tex

```diff
@@ -3203,6 +3205,44 @@ \section{Ternary Operator}
 And hence, the ternary operator is accepted in such cases.
 
 
+\section{Backslashes}
+\label{sec:Backslashes}
+
+The use of a backslash as line continuation character appears to sometimes
+hamper code readability. A backslash at the end of a line that has C++ \code{//}
+comment may be quite confusing. For example, it may be overlooked easily that
+the following code has commented out the \code{Sleep()} function call:
+
+\small
+\begin{minted}[baselinestretch=1,fontsize=\footnotesize,linenos=false,bgcolor=ltgray]{cpp}
+  // Wil we go to sleep or not? \
+  Sleep();
+\end{minted}
+\normalsize
```

Comment:
Thanks for your feedback. My intention was to show two different cases for which the use of backslashes is problematic. In order to reach the conclusion that we discourage them _in general_ (except for in macro definitions), in ITK C++ source code.

However, we could also just have a guideline _specifically_ against backslashes in string literals. Without mentioning other possible cases. Is that what you prefer?

Reply:
Although in other places the "recommended", "suggested", etc. words have been used (and this has also caused some debate), IMHO, in this case, as I said, a commit containing backslashes is unlikely to get accepted following your PRs in ITK, so to me, following your PRs and the improvement in style/message rendering they bring, "ITK does not use backslashes to break continuous string literals into multiple lines; it relies on its format enforcing tool -name it and cross ref- to break them." (or something similar with the same message; and then the macro stuff would come.

Reply:
Thanks for your comments Jon, but...

>  a commit containing backslashes is unlikely to get accepted following your PRs in ITK

If that's the case, we might as well abandon this PR, right?

I thought it would be helpful to make it a guideline, because I encountered quite a few of those backslashes before doing ITK PR https://github.com/InsightSoftwareConsortium/ITK/pull/3713. But if we may assume that those backslashes would not be accepted anymore now anyway, the guideline may not be necessary.

Reply:
As you feel it Niels.